integration-tests/contract-security-zero-transfer.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { CONFIGS } from './config';
+import { RpcClient } from '@taquito/rpc';
+
+// TC-007 - A 0tez transaction to an implicit account should fail.
+
+const client = new RpcClient(' https://ithacanet.ecadinfra.com');
```

Comment:
client doesn't seem to be used and can be removed

Reply:
removed. thanks